membership_install/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "Community Builder",
+    "summary": "Tools to build your community",
+    "author": "Onestein",
+    "website": "https://onestein.nl",
+    "category": "Membership",
+    "version": "16.0.1.0.0",
+    "license": "AGPL-3",
+    "depends": [
+        "membership",
+        "membership_accessibility",
+        "membership_committee",
+        "membership_section",
+        "membership_contribution",
+        "membership_extension",
+        "membership_origin",
```

Comment:
We are going to remove this module as well as membership_report and add tags on members, so this should be excluded I think or we take care of this when membership changes are merged?